vrk-kpa/suomidigi
# Suopa-X_twitter_posts_styling

**To test:** 

- make fresh
- Edit the front page (Click the Edit button under the headline “Suomidigi”)
- Add a new Paragraph type “Add View”
- Choose “Social Media” from the View drop down menu
- Choose “Twitter” from the Display drop down menu
- Save
- Go to Structure > Social media types > Clear social media entities and rerun the importer
- You should see a message “The importer has been run and social media posts has been updated.” if everything went smoothly. 
- Go back to front page (https://suomidigi.docker.sh/en)
- Flush all caches
- You should now see the Twitter posts on the page.
- Check that the layout of the Twitter posts looks like this: https://xd.adobe.com/view/c1b40caa-b2e0-43a5-78d8-9690a43f540c-b785/screen/7c1dc8eb-0b46-4aa5-b434-be0fbcd8fc0c/Suomidigi-etusivu-launch/
- Click some of the Twitter posts > the post should open in a new tab. 

@khalima
